Dockerfile

```diff
@@ -38,7 +38,8 @@ ENV VERDACCIO_APPDIR=/opt/verdaccio \
     VERDACCIO_USER_NAME=verdaccio \
     VERDACCIO_USER_UID=10001 \
     VERDACCIO_PORT=4873 \
-    VERDACCIO_PROTOCOL=http
+    VERDACCIO_PROTOCOL=http \
+    VERDACCIO_ADDRESS=[::]
```

Comment:
Thinking of maybe the previous one 0.0.0.0 should be default and this one by demand instead at least until next major release.

Thoughts?

Reply:
I'd agree with that, I only did it this way to avoid having it switch back and forth for other users. I can update the default to 0.0.0.0 and add a note in the docs to show how to set the dual-stack option.